Classes/Service/SettingsService.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Skyfillers\SfSimpleFaq\Service;
+namespace SKYFILLERS\SfSimpleFaq\Service;
```

Comment:
Please leave the namespace in uppercase. We will adjust this maybe later in a separate patch.